app/openlxp_xds_project/urls.py

```diff
@@ -17,8 +17,16 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path
+import django_saml2_auth.views
 
 urlpatterns = [
+    # These are the SAML2 related URLs. You can change "^saml2_auth/" regex to
+    # any path you want, like "^sso_auth/", "^sso_login/", etc. (required)
+    path('saml2_auth/', include('django_saml2_auth.urls')),
+    # The following line will replace the admin login with SAML2 (optional)
+    # If you want to specific the after-login-redirect-URL, use parameter "?next=/the/path/you/want"

```

Comment:
Nit: specific -> specify

Reply:
woops, the dangers of copy-pasting documentation.